cp-ksql-server/include/etc/confluent/docker/launch

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+#
+# Copyright 2017 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# JMX settings
+if [ -z "$KSQL_JMX_OPTS" ]; then
+  KSQL_JMX_OPTS="-Dcom.sun.management.jmxremote -Dcom.sun.management.jmxremote.authenticate=false -Dcom.sun.management.jmxremote.ssl=false "
+fi
+
+# The JMX client needs to be able to connect to java.rmi.server.hostname.
+# The default for bridged n/w is the bridged IP so you will only be able to connect from another docker container.
+# For host n/w, this is the IP that the hostname on the host resolves to.
+
+# If you have more that one n/w configured, hostname -i gives you all the IPs,
+# the default is to pick the first IP (or network).
+export KSQL_JMX_HOSTNAME=${KSQL_JMX_HOSTNAME:-$(hostname -i | cut -d" " -f1)}
+
+# JMX port to use
+if [  $KSQL_JMX_PORT ]; then
+  export JMX_PORT=$KSQL_JMX_PORT
+export KSQL_JMX_OPTS="$KSQL_JMX_OPTS -Djava.rmi.server.hostname=$KSQL_JMX_HOSTNAME -Dcom.sun.management.jmxremote.local.only=false -Dcom.sun.management.jmxremote.rmi.port=$JMX_PORT -Dcom.sun.management.jmxremote.port=$JMX_PORT"
+fi
+
```

Comment:
we probably need to also export the `KSQL_LOG4J_OPTS` environment variable to point to the right log4j.properties file.

ie. we need something like `KSQL_LOG4J_OPTS=/etc/ksql-server/log4j.properties`. Otherwise KSQL will look in a predefined path relative to the run script, which is wrong in this case.

We also should specify `KSQL_CONFIG_DIR=/etc/ksql-server` since the config dir is in a non-standard location.